README.md

```diff
@@ -59,16 +59,17 @@ Refer to [`direnv` and `devx`](./docs/direnv.md) guide for more information.
 ## VSCode DevContainer / GitHub CodeSpace support
 
 To make this developer shell available in VSCode DevContainer or GitHub CodeSpace, simply add a file named `.devcontainer/devcontainer.json` with the following content:
-```json
+```jsonc
```

Comment:
```suggestion
```json
```

Reply:
I think this should be json only?

Reply:
No, because that's the format that use VSCode (JSON that supports comments). If using `json` the code block will highlight the comment in red within GitHub interface (as it will be considered as a syntax error).